.changeset/dark-circles-whine.md

```diff
@@ -0,0 +1,12 @@
+---
+"@fluidframework/tree": minor
+---
+---
+section: tree
+---
+
+Exposes the view schema from the TreeView interface.
```

Comment:
Nits; I think it reads better in release notes, and we skip trailing stop for headings
```suggestion
Expose the view schema from the TreeView interface
```